### PR TITLE
Style add habit screen

### PR DESCRIPTION
**Pull request by devinjameson**

### CODE REVIEW

File: src/Home.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const HomeScreen: FunctionComponent<HomeScreenProps> = observer(({ store }) => {
   }, [store])
 
   const habitIds = Object.keys(store.habits)
+  const reversedHabitIds = [...habitIds].reverse()
```
Comment (johnschoeman) [inline src/Home.tsx:36]: This is an interesting one.  Given that we have many habits now, we might need to spend some time thinking of sort order and wether or not the user should choose the order.

File: src/Home.tsx [hunk 2]
```diff
@@ -48,7 +49,7 @@ const HomeScreen: FunctionComponent<HomeScreenProps> = observer(({ store }) => {
         }}
         contentContainerStyle={style.contentContainer}
       >
-        {habitIds.map(toHabitListItem)}
+        {reversedHabitIds.map(toHabitListItem)}
```
Comment (johnschoeman) [inline src/Home.tsx:52]: A bit of a naming philosophy thought: The fact that these are 'reversed' doesnt seem to be relevant to this component as the order isnt a first class domain concept. (reverse relative to what? time? created_at?, alphabetical?). Therefore we shouldnt have this in the name. what do you think about:

```typescript
const habitIds = [...Object.keys(store.habits)].reverse()
...
{habitIds.map(toHabitListItem)}
```
for now and we can make the sorting logic first class in a later commit to something like:
```typescript
const habits = HabitStore.byLastCompleted(store.habits)
const habitIds = habits.map(habit => habit.id)
```

**Review by johnschoeman — COMMENTED** [review]
Nice work!

File: src/Home.tsx (hunk 2 above)
Comment (devinjameson) [reply src/Home.tsx:52]: Makes sense to m!

**Pull request merged**